packages/calcite-components/src/demos/action.html

```diff
@@ -117,7 +120,7 @@
         <div class="child right-aligned-text">with svg</div>
 
         <div class="child">
-          <calcite-action indicator active text="click-me" label="hello world" text-enabled scale="s">
+          <calcite-action indicator type="toggle" pressed text="click-me" label="hello world" text-enabled scale="s">
```

Comment:
This is an example where setting `toggled` (or "a single active prop" per my other comment) would make more sense IMO as an implementer.